### ISSUE TITLE
footer links have insufficient contrast

### ISSUE THREAD

**Issue by TheTeaCat** [TheTeaCat/lastfm-tag-cloud#37]

**Comment by TheTeaCat** [comment]
71dc3f67749741128590aa817ffaa1824d5ce0b3 closes this.
